samples/snippets/storage_download_to_stream.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START storage_download_to_stream]
+from google.cloud import storage
+
+
+def download_blob_to_stream(bucket_name, source_blob_name, file_obj):
```

Comment:
I believe this sample is backward. It should be streaming the file from cloud storage to a local file.

Reply:
Not sure I understand, will follow up in chat

Reply:
Okay, sounds like we're on the same page now.

Reply:
nit: Would you mind updating the file names to match the region tags? Not required, just easier to find and maintain the snippet files.

Reply:
Can do

Reply:
Oh, hmm, actually the file names are uniform but the region tags are not -- I'll ping you in chat